apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/json/InputFieldJsonWriter.kt

```diff
@@ -80,6 +80,7 @@ class InputFieldJsonWriter(
       is GraphQLString -> writeString(fieldName, customTypeValue.value)
       is GraphQLBoolean -> writeBoolean(fieldName, customTypeValue.value)
       is GraphQLNumber -> writeNumber(fieldName, customTypeValue.value)
+      is GraphQLNull -> writeString(fieldName, null)
```

Comment:
nit: should we introduce `writeNull` to our `JsonWriter`? 😄

Reply:
`JsonWriter` already has `nullValue`, would that be ok? 

We could also add `writeNull` to `InputFieldWriter` but that would be a much more invasive change that I'm not too keen on doing in 2.x. Falling into the rabbit hole of `InputFieldsWriters`, I realized that `SortedInputFieldMapWriter` doesn't handle custom scalar types. So custom scalars as field arguments will most likely fill the cache with different values. I guess something to keep in mind for `3.x`